scripts/docstrings_checker.py

```diff
@@ -174,7 +174,9 @@ def docstringify(docstring):
 
 
 class GoogleDocstring(_check_docs_utils.GoogleDocstring):
-
+    """Class that checks whether docstrings follows Google Python Style Guide
```

Comment:
that checks --> for checking (the class itself doesn't do the checking, I think)
follows --> follow
Google Python Style Guide or not. --> the Google Python Style Guide.